lib/nfc/protocols/st25tb/st25tb_poller.h

```diff
@@ -14,6 +14,7 @@ typedef struct St25tbPoller St25tbPoller;
 typedef enum {
     St25tbPollerEventTypeError,
     St25tbPollerEventTypeReady,
+    St25tbPollerEventTypeReadSuccessful,
 } St25tbPollerEventType;
```

Comment:
I am not sure that both Ready and ReadSuccess events are necessary here. For example, if we take Iso14443-3a protocol, there is only Ready event, because there are some protocols based on Iso14443-3a protocol. And if we take Mifare Ultralight protocol, it runs state machine on Iso14443-3a Ready event and spawns ReadSuccess or ReadFail events.
Since there are no other protocols based on St25tb poller, I would leave just ReadSuccess event.

Reply:
The Ready event is mainly used by the sync api in order to not have to wait for a full tag read before executing commands in case eg detect type or read block is called